packages/editor/src/lib/editor/Editor.ts

```diff
@@ -1451,39 +1537,27 @@ export class Editor extends EventEmitter<TLEventMap> {
 	 * @example
 	 * ```ts
 	 * editor.updateCurrentPageState({ id: 'page1', editingShapeId: 'shape:123' })
-	 * editor.updateCurrentPageState({ id: 'page1', editingShapeId: 'shape:123' }, { ephemeral: true })
 	 * ```
 	 *
 	 * @param partial - The partial of the page state object containing the changes.
-	 * @param historyOptions - The history options for the change.
 	 *
 	 * @public
 	 */
 	updateCurrentPageState(
 		partial: Partial<
 			Omit<TLInstancePageState, 'selectedShapeIds' | 'editingShapeId' | 'pageId' | 'focusedGroupId'>
-		>,
-		historyOptions?: TLHistoryBatchOptions
+		>
 	): this {
-		this._updateCurrentPageState(partial, historyOptions)
+		this._updateCurrentPageState(partial)
 		return this
 	}
-	_updateCurrentPageState = (
-		partial: Partial<Omit<TLInstancePageState, 'selectedShapeIds'>>,
-		historyOptions?: TLHistoryBatchOptions
+	private _updateCurrentPageState = (
```

Comment:
Now that `updateCurrentPageStage` and `_updateCurrentPageShape` are the same, lets remove the `_` and just keep the main one. We don't need both.

Also, I'm nervous about this change. Having these defaults is pretty convenient, and this is silently introducing bugs for anyone updating to the next version unless they notice this change. We can call it out in the release notes, but i'm still not sure that would be enough...

Reply:
3.0?

Reply:
yeah the page state changes should be ignored in all cases except updating selectedShapeIds, which there's a separate method for. I think we should revert this to include the historyOptions and use ignore by default.